tests/testsuite/cargo_command.rs

```diff
@@ -245,32 +245,6 @@ fn displays_subcommand_on_error() {
         .run();
 }
 
-#[cargo_test]
-fn override_cargo_home() {
```

Comment:
This test should probably be kept, but changed in such a way that it still works.  It is one of the only tests that verifies config lookup with an overridden `CARGO_HOME`.  

Maybe instead of checking the name/email, it could check that a `.git` directory was not created (as part of `git = false`).  Then run `cargo new` a second time with `CARGO_HOME` unset and verify it does create a `.git` directory.

Reply:
I found the test was broken and was not using the `vcs` option to begin with. Fixed that and rewrote as suggested.